packages/scoutgame-ui/src/components/claim/components/PointsClaimContainer.tsx

```diff
@@ -16,21 +19,30 @@ export async function PointsClaimContainer() {
     return null;
   }
 
-  const [err, data] = await safeAwaitSSRData(getClaimablePointsWithSources(scoutId));
+  const platform = getPlatform();
+
+  const isOnchainApp = platform === 'onchainwebapp';
+
+  const [err, data] = await safeAwaitSSRData(
+    (isOnchainApp ? getClaimableTokensWithSources : getClaimablePointsWithSources)(scoutId)
+  );
 
   if (err) {
     return null;
   }
 
   const { bonusPartners, points, builders, repos } = data;
 
+  const claimData = isOnchainApp ? (data as UnclaimedTokensSource).claimData : undefined;
+
   return (
     <>
       <PointsClaimScreen
         totalUnclaimedPoints={points}
         bonusPartners={bonusPartners}
         builders={builders}
         repos={repos}
+        claimData={claimData}
```

Comment:
we should probably add 'onchain' prefix to any prop that is specifically for onchain stuff